pkg/ansible/proxy/proxy.go

```diff
@@ -44,6 +44,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/source"
 )
 
+const AutoSkipCacheRE = "^/api/.*/pods/.*/exec,^/api/.*/pods/.*/attach"
```

Comment:
```suggestion
const AutoSkipCacheREList = "^/api/.*/pods/.*/exec,^/api/.*/pods/.*/attach"
```

Reply:
Can these be pre-compiled?

Reply:
currently they're only compiled once, when the proxy starts, I don't think there's a safe way to compile them into a const though, compiling the regex either returns `regexp, err` or panics AFAICT

Reply:
Since its a list you'd have to do some annoying stuff with an `init` function, so the way you have it now is fine.